package.json

```diff
@@ -31,33 +32,39 @@
   "license": "CC0-1.0",
   "devDependencies": {
     "@11ty/eleventy": "^2.0.1",
-    "@11ty/eleventy-img": "^1.1.0",
     "@11ty/eleventy-navigation": "^0.3.5",
     "@11ty/eleventy-plugin-rss": "^1.2.0",
     "@11ty/eleventy-plugin-syntaxhighlight": "^5.0.0",
-    "autoprefixer": "^10.4.16",
     "check-html-links": "^0.2.4",
     "chokidar-cli": "^3.0.0",
     "eleventy-plugin-svg-sprite": "^1.3.1",
-    "esbuild": "^0.19.4",
-    "esbuild-sass-plugin": "^2.16.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-import": "^2.29.0",
     "html-validate": "^8.7.0",
     "js-yaml": "^4.1.0",
-    "luxon": "^2.3.1",
     "markdown-it": "^12.3.2",
     "markdown-it-anchor": "^8.6.7",
     "markdown-it-attrs": "^4.1.6",
+    "netlify-cms": "^2.8.0",
     "netlify-cms-proxy-server": "^1.3.23",
     "npm-run-all": "^4.1.5",
-    "pa11y-ci": "^3.0.1",
+    "pa11y-ci": "^2.4.2",
     "postcss": "^8.4.31",
     "postcss-cli": "^9.1.0",
+    "prettier": "3.0.3",
     "rimraf": "^3.0.2",
     "start-server-and-test": "^2.0.1"
   },
   "dependencies": {
+    "@11ty/eleventy-img": "^1.1.0",
     "@uswds/uswds": "3.6.1",
+    "autoprefixer": "^10.4.16",
+    "esbuild": "^0.19.4",
+    "esbuild-sass-plugin": "^2.16.0",
+    "luxon": "^2.3.1",
     "markdown-it-footnote": "^3.0.3",
-    "netlify-cms": "^2.10.192"
+    "netlify-cms": "^2.10.192",
+    "pa11y-ci": "^3.0.1"
```

Comment:
[blocking question] @eric-gade I see a bunch of dev dependencies were moved to production, including pa11y (pa11y's now under both). Can they be moved back to dev?

Reply:
I've gone ahead and arranged what (I think) it should be in b72e5c1, thanks